data/data/vsphere/master/main.tf

```diff
@@ -18,16 +19,18 @@ resource "vsphere_virtual_machine" "vm" {
   }
 
   disk {
-    label = "disk0"
-    size  = var.disk_size
+    label            = "disk0"
+    size             = 120
+    eagerly_scrub    = var.scrub_disk
+    thin_provisioned = var.thin_disk
   }
 
   clone {
     template_uuid = var.template
   }
 
   extra_config = {
-    "guestinfo.ignition.config.data"          = base64encode(data.ignition_config.ign[count.index].rendered)
+    "guestinfo.ignition.config.data"          = base64encode(data.ignition_config.ign[count.index].id)
```

Comment:
@patrickdillon `.id` here needs to stay `.rendered`

Reply:
Do you not encounter this error
```
ERROR Error: Unsupported attribute                 
ERROR                                              
ERROR   on ../../../../../../../../tmp/openshift-install-429505789/master/ignition.tf line 25, in data "ignition_config" "ign": 
ERROR   25:     data.ignition_file.hostname[count.index].rendered 
ERROR     |----------------                        
ERROR     | count.index is 2                       
ERROR     | data.ignition_file.hostname is tuple with 3 elements 
ERROR                                              
ERROR This object does not have an attribute named "rendered". 
ERROR      
``` 
or is there another solution? 

I assume the change should be reverted here as well: https://github.com/openshift/installer/pull/2938/files#diff-faa0636ede0a1934d8c386673a6fb84eR25?

Reply:
I reverted my change to `.rendered` in both files. Leaving it as `.rendered` gives me an error, but I don't want to block the rest of the pr on that. We can add it later.